Foundation/NSError.swift

```diff
@@ -670,6 +670,43 @@ public struct CocoaError : _BridgedStoredNSError {
     }
 }
 
+internal extension CocoaError {
+    static let errorMessages = [
+        Code(rawValue: 4): "The file doesn’t exist.",
```

Comment:
Can you open a bug on https://bugs.swift.org to localize these?

Reply:
Sure

Reply:
@millenomi https://bugs.swift.org/browse/SR-10523